### ISSUE TITLE
There is no NLP in this course 

### ISSUE THREAD

**Issue by Khalid-Siddiqi** [mrdbourke/pytorch-deep-learning#1104]
Is there any possibility to get NLP in this course

**Comment by mrdbourke** [comment]
Yes! That is a great idea, currently planning expansions for 2025 and NLP is on the list.

In the meantime, you can try Learn Hugging Face for an NLP example: https://www.learnhuggingface.com/notebooks/hugging_face_text_classification_tutorial
